cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,53 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('/alerts');
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
 
+    cy.clickButton('#alertButton');
+    cy.on('window:alert', (message) => {
+      expect(message).to.equal('You clicked a button');
+    });
   });
 
   it('should have the ability to assert scheduled allert', () => {
 
+    cy.clickButton('#timerAlertButton');
+    cy.wait(5000);
+    cy.on('window:alert', (message) => {
+      expect(message).to.equal('This alert appeared after 5 seconds');
+    });
   });
 
   it('should autimatically resolve alerts', () => {
+    cy.clickButton('#confirmButton');
+    cy.on('window:confirm', (message) => {
+      expect(message).to.equal('Do you confirm action?');
+    });
 
+    cy.confirmText('#confirmResult', 'You selected Ok');
   });
+  
+  it('should have the ability to cancel alerts', () => {
+    cy.clickButton('#confirmButton');
 
-  it('should have the ability to Cancel alerts', () => {
+    cy.on('window:confirm', (message) => {
+      expect(message).to.equal('Do you confirm action?');
+      return false;
+    });
 
+    cy.confirmText('#confirmResult', 'You selected Cancel');
   });
 
   it('should have the ability to enter text to alert', () => {
+    const message = 'Test';
+
+    cy.window().then((win) => {
+      cy.stub(win, 'prompt').returns(message);
+    });
 
+    cy.clickButton('#promtButton');
```

Comment:
There is a typo in the button ID. It should be '#promptButton' instead of '#promtButton'.